src/include/ZividPython/DataModelWrapper.h

```diff
@@ -292,6 +290,64 @@ namespace ZividPython
             }
             return pyClass;
         }
+
+        constexpr bool isDirectChild(const std::string_view parent, const std::string_view child)
+        {
+            // Top-level nodes are always direct children of the root
+            if(parent.empty() && child.find('/') == std::string::npos)
+            {
+                return true;
+            }
+            if(child.size() <= parent.size())
+            {
+                return false;
+            }
+            if(child.compare(0, parent.size(), parent) != 0)
+            {
+                return false;
+            }
+            // Ensure there are no additional path components
+            const auto remainingPath = child.substr(parent.size());
+            return remainingPath.find('/') == 0 && remainingPath.find('/', 1) == std::string::npos;
+        }
+
+        static_assert(isDirectChild("", "TopLevel") == true);
+        static_assert(isDirectChild("", "TopLevel/WithChild") == false);
+        static_assert(isDirectChild("TopLevel", "TopLevel/WithChild") == true);
+        static_assert(isDirectChild("TopLevel", "TopLevel/WithChild/GrandChild") == false);
+
+        template<typename DM, typename PyDest, typename Node>
+        void findAndWrapUninstantiatedNodesInDataModel(PyDest &dest, const Node &node)
+        {
+            node.forEach([&](const auto &member) {
+                using MemberType = std::remove_const_t<std::remove_reference_t<decltype(member)>>;
```

Comment:
maybe you also need to remove `volatile` qualifier, so `std::remove_cv_t` instead of `std::remove_const_t`.

Reply:
I guess I could also use `decay_t`

Reply:
that works too, but I generally avoid it as it also performs array -> pointer conversion. That shouldn't be an issue in this case though.

Reply:
Updated with `remove_cv_t`. I noticed that C++20 actually has a `remove_cvref_t` that we might be able to use in the future :)